test/services/edusharing/index.test.js

```diff
@@ -0,0 +1,12 @@
+const { expect } = require('chai');
```

Comment:
Codacy found an issue: ['expect' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4977295)